kong/plugins/jwt/handler.lua

```diff
@@ -24,19 +23,21 @@ JwtHandler.VERSION = "2.1.0"
 -- @param conf Plugin configuration
 -- @return token JWT token contained in request (can be a table) or nil
 -- @return err
-local function retrieve_token(conf)
+local function retrieve_tokens(conf)
```

Comment:
Probably you want for backwards compatibility to keep `retrieve_token` around, which would call `retrieve_tokens` and return the first one, or something.